source/includes/_async_jobs.md

```diff
@@ -0,0 +1,156 @@
+|**completedAt:**  |**string, optional**|
+|                  |When this job completed. There's a `result` or `error` if this is present. Is an **ISO 8601** encoded date. Can be null if it hasn't finished yet.|
+|**error:**        |**string, optional**|
+|                  |An error message that indicates a problem when processing the job. There is no `result` if this is present.|
+|**result:**       |**string, optional**|
+|                  |The final result from processing this job. What it is depends on job `kind`.|
+|**partnerApp:**   |**string, optional**|
+|                  |The ID of the Partner App that created this Async Job. Included only when authorization is made with the API key.|
+
+## Get Async Job
+
+Returns the requested async job. When using OAuth, only jobs started by the authenticated app can be accessed.
+
+```shell
+# EXAMPLE REQUEST
+$ curl "https://app.conversio.com/api/v1/async-jobs/57b5aa3b046abfb053d80b52" \
+  -H "X-ApiKey: YOUR_API_KEY" \
+  -H "Accept: application/json" \
+  -X GET
+```
+
+> EXAMPLE RESPONSE
+
+```json
+{
+  "data": {
+    "id": "57b5aa3b046abfb053d80b52",
+    "kind": "newsletter-recipients",
+    "status": "done",
+    "startedAt": "2017-01-03T16:32:27.741Z",
+    "completedAt": "2017-01-03T16:37:20.417Z",
+    "result": "https://app.conversio.com/async-jobs/57b5aa3b046abfb053d80b52.csv"
+  }
+}
+```
+
+### Get an Async Job [GET]
+
+`https://app.conversio.com/api/v1/async-jobs/{ASYNC_JOB_ID}`
+
+_OAuth Scopes_: read_async_jobs
+
+<aside class="success">
+  Response 200
+</aside>
+
+<aside class="warning">
+  Response 404 - Async Job not found. One of the following happened:
+  <ul>
+    <li>The given ID does not belong to an existing job</li>
+    <li>The ID belongs to another shop's job</li>
+    <li>The ID belongs to another Partner App's job</li>
+    <li>The job has expired</li>
+  </ul>
+</aside>
+
+### Response Body
+
+Then endpoint returns an object with a `data` key that is itself an object with the Async Job's properties. It will include the Partner App if authorized with an API key. The following keys are returned:
+
+|Key               |Details    |
+|-----------------:|-----------|
+|**id:**           |**string**|
+|                  |The Async Job's ID. You used it to call this endpoint.|
+|**kind:**         |**string**|
+|                  |What kind of job this is. Currently only supports "newsletters-recipients".|
+|**status:**       |**string**|
+|                  |The job's status. One of "pending", "done" or "failed".|
+|**startedAt:**    |**string, optional**|
+|                  |When this job started processing. Is an **ISO 8601** encoded date. Can be null if it hasn't started yet.|
+|**completedAt:**  |**string, optional**|
+|                  |When this job completed. There's a `result` or `error` if this is present. Is an **ISO 8601** encoded date. Can be null if it hasn't finished yet.|
```

Comment:
Can this be undefined. The reason I ask is, you have stated: 'if this is present'.

Also if you are changing the one above from can -> will then this one needs some love too